guides/source/active_storage_overview.md

```diff
@@ -858,12 +868,6 @@ a link to a lazily-generated preview, use the attachment's [`preview`][] method:
 To add support for another format, add your own previewer. See the
 [`ActiveStorage::Preview`][] documentation for more information.
 
-WARNING: Extracting previews requires third-party applications: FFmpeg v3.4+ for
-video and muPDF for PDFs, and on macOS also XQuartz and Poppler.
```

Comment:
I do not use macOS...  Is XQuartz no longer required?  Or can we just assume it's installed already?

Reply:
Oh yes, I forgot to add a comment about this one. XQuartz was not in the original list of dependencies for AST. It was added in 0af43ac7f6583afd3400c176fde88b96af58dc77. I have no idea why. The homebrew formulas for installing ffmpeg, mupdf and poppler do not have it.

Reply:
I've also been using both ffmpeg and poppler constantly in my mac and they never complained about XQuartz.

Reply:
Hmm, indeed both ["mupdf" "xquartz"](https://www.google.com/search?q=%22mupdf%22+%22xquartz%22) and ["poppler" "xquartz"](https://www.google.com/search?q=%22poppler%22+%22xquartz%22) yield very few results.  If XQuartz is a transitive dependency of either MuPDF or Poppler, perhaps we can assume that it will be taken care of as part of properly installing those tools.